nubis/files/confd/templates/sfmc-fetcher.tmpl

```diff
@@ -0,0 +1,4 @@
+config = {
+  'api_key': '{{ getv "/config/SalesforceSFMC/APIKey" }}'
```

Comment:
In general, it's preferrable  to stick a default value in there,  otherwise, if the value doesn't exist in Consul, the confd template will just hard *fail*

```
  'api_key': '{{ getv "/config/SalesforceSFMC/APIKey" "api-key-not-set" }}'
```